src/devices/Mhz19b/samples/Mhz19b.Sample.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Iot.Device.Mhz19b.Samples
+{
+    /// <summary>
+    /// Samples for Mhz19b
+    /// </summary>
+    public class Program
+    {
+        /// <summary>
+        /// Main entry point
+        /// </summary>
+        public static void Main(string[] args)
+        {
+            Mhz19b sensor = new Mhz19b("/dev/serial0");
```

Comment:
Maybe add something for the calibration to show how to use it?

Reply:
I could add this as code commented-out. Altering any of the calibration settings could make the device unusable for a user who can't perform a real calibration. I don't want to let someone run accidentally into this.

Reply:
Yes, that's an option or then just into the Readme.md in this case. So someone advance who want to calibrate will have the correct few needed lines.

Reply:
Examples added.